nucypher/cli/literature.py

```diff
@@ -594,3 +594,26 @@
 PORTER_BOTH_TLS_KEY_AND_CERTIFICATION_MUST_BE_PROVIDED = "Both --tls-key-filepath and --tls-certificate-filepath must be provided to launch porter with TLS; only one specified"
 
 PORTER_BASIC_AUTH_REQUIRES_HTTPS = "Basic authentication can only be used with HTTPS. --tls-key-filepath and --tls-certificate-filepath must also be provided"
+
+
+#
+# PREApplication
+#
+
+STAKING_PROVIDER_UNAUTHORIZED = '{provider} is not authorized.'
+
+CONFIRM_BONDING = 'Are you sure you want to bond staking provider {provider} to operator {operator}?'
+
+BONDING_TIME = 'Bonding not permitted until {date}'
```

Comment:
This is used for both cases right, as part of `time_elapsed`.

```suggestion
BONDING_TIME = 'Bonding/unbonding not permitted until {date}'
```

Reply:
✔️